lib/persisted-model.js

```diff
@@ -897,9 +897,10 @@ module.exports = function(registry) {
         description: 'Get a set of deltas and conflicts since the given checkpoint.',
         accessType: 'READ',
         accepts: [
-          {arg: 'since', type: 'number', description: 'Find deltas since this checkpoint'},
-          {arg: 'remoteChanges', type: 'array', description: 'an array of change objects',
-            http: {source: 'body'}},
+          { arg: 'since', type: 'number', description: 'Find deltas since this checkpoint',
+            http: { source: 'form' }},
```

Comment:
This is a breaking change which we cannot land in `3.x` version.

I am proposing to keep `remoteChanges` mapped to `body`, and explicitly map `since` to a query string argument (I think that was the initial intention anyways). To prevent breaking change in the mapping of the `since` argument, there should be a model setting to control whether the old or the fixed mapping is used.